sig-docs/charter.md

```diff
@@ -0,0 +1,97 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs publishes Kubernetes documentation on kubernetes.io. Kubernetes documentation includes:
+  - Documentation of the core Kubernetes APIs
+  - Core Kubernetes architecture 
+  - CLI tools shipped with the Kubernetes release
+
+ Responsibility for creating feature documentation belongs to the developers and SIGs associated with a feature. This includes documentation for features, concepts, components, opinions, processes, etc.
```

Comment:
Reword to be more explicit, and remove mention of "opinions, processes, etc". : Responsibility for creating feature documentation belongs to the developers and SIGs creating each feature. This includes task-driven documentation for the feature itself and conceptual documentation about the feature.